crypto/evp/m_sha3.c

```diff
@@ -134,7 +134,234 @@ static int shake_ctrl(EVP_MD_CTX *evp_ctx, int cmd, int p1, void *p2)
     }
 }
 
-#define EVP_MD_SHA3(bitlen)                     \
+#if defined(OPENSSL_CPUID_OBJ) && defined(__s390__) && defined(KECCAK1600_ASM)
+/*
+ * IBM S390X support
+ */
+# include "../s390x_arch.h"
+
+# define S390X_SHA3_FC(ctx)	((ctx)->pad)
+
+# define S390X_sha3_224_CAPABLE	((OPENSSL_s390xcap_P.kimd[0] &		\
+                                  S390X_CAPBIT(S390X_SHA3_224)) &&	\
```

Comment:
whats wrong with the tabs here ? i mean the line breaks do not line up here, because patch additions "+" prefix which happens to hit a "tab boundary" in the first line

Reply:
I understand why they don't line up, but it's not it. Styling guide says "don't use tabs".

Reply:
ok i thought this only applied to identation because its 4 spaces. So if it means "no tabs at all", even before the macro line break, i have more places to fix